llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -952,7 +641,8 @@ void FilterChooser::dumpStack(raw_ostream &OS, indent Indent,
 // Inst{20} = 1 && Inst{3-0} == 0b1111 represents two islands of yet-to-be
 // decoded bits in order to verify that the instruction matches the Opcode.
 std::vector<FilterChooser::Island>
-FilterChooser::getIslands(const KnownBits &EncodingBits) const {
+FilterChooser::getIslands(const KnownBits &EncodingBits,
```

Comment:
I'll extract this into separate PR. Not that it would make the diff much smaller, but this change is separable.